app/views/applicant_details/_form.html.erb

```diff
@@ -71,7 +71,7 @@
   <div class="field">
     <%= form.label :birthdate%>
     <div class="date_select">
-      <%= form.date_select :birthdate, start_year: Date.today.year - 19, end_year: Date.today.year - 12, prompt: "Select",required: true %>
+      <%= form.date_select :birthdate, default: applicant_detail.birthdate, start_year: Date.today.year - 19, end_year: Date.today.year - 12, prompt: "Select",required: true %>
```

Comment:
I am not understanding why the default: applicant_detail.birthdate is needed. This is pulled with the applicant_detail before action.

Reply:
@rsmoke Because, if you edit a user's applicant_detail record you want to see here the user's birthday and not empty values like in a new form. 
form.date_select creates three input fields and doesn't keep the saved data without default:
![Screenshot 2024-01-02 at 3 53 37 PM](https://github.com/lsa-mis/mmss-mysql/assets/56089936/d4dbf762-2508-4e30-bc88-c9db3e7fa2b6)
Please try to edit someone's applicant_details.